urban:
  husky1:
    x: -3.305017984993041
    y: 0.4451560487041896
    z: 0.1289591814436012

    qx: 0.0020204908706928236
    qy: 0.0054744630398248185
    qz: -0.04137968394028248
    qw: 0.9991264385647961

    roll: 0.003584617370095291
    pitch: 0.011106804423871578
    yaw: -0.08276450599004169

  husky4:
    x: -3.05828163233025
    y: 1.3146063057792554
    z: 1.3146063057792554

    qx: -0.0015753973838351385
    qy: 0.0001138999980628975
    qz: -0.12098729555857027
    qw: 0.9926530256283129

    roll: -0.0031552121378665638
    pitch: -0.00015507978302973689
    yaw: -0.24256896090382354

  spot1:
    x: -3.3300443667943584
    y: -0.40779892931947404
    z: 0.09902670291861995

    qx: -0.004348854190418861
    qy: -0.018849932192913487
    qz: 0.049967535322706064
    qw: 0.9985635777094882

    roll: -0.010576506154311873
    pitch: -0.037219701261237026
    yaw: 0.10019229309426517
